data.js

```diff
@@ -1293,6 +1297,11 @@ var data = [
         value: 'Security Information and Event Management',
         explanation: "Security information and event management is a subsection within the field of computer security, where software products and services combine security information management and security event management. They provide real-time analysis of security alerts generated by applications and network hardware."
     },
+    {
+        label: 'SLA',
+        value: 'Service Level Agreement',
+        value_fr: 'ANS: accord sur les niveaux de service' 
```

Comment:
Hi Nisa, you should put ANS under label_fr and have value_fr: "Accord sur les Niveaux de Service"